src/Resources/contao/drivers/DC_Table.php

```diff
@@ -1753,7 +1753,7 @@ public function move()
 							   ->execute($row[1]['sorting'], $row[0]['id']);
 
 				// Invalidate cache tags
-				$this->invalidateCacheTags($this->getCacheTags($this->strTable, array($row[1]['id'], $row[0]['id'])));
+				$this->invalidateCacheTags($this->getCacheTags($this->strTable, array($row[1]['id'], $row[0]['id']), (string) $this->ptable, (int) $this->activeRecord->pid));
```

Comment:
Why did you cast the values? The `invalidateCacheTags()` method seems to handle numeric strings as parent ID and there are no type-hints that would enforce a certain type.